src/preact/base-element.js

```diff
@@ -194,6 +194,11 @@ export class PreactBaseElement extends AMP.BaseElement {
       ...templatesInit,
     });
 
+    const staticProps = Ctor['staticProps'];
+    if (staticProps) {
+      Object.assign(/** @type {!Object} */ (this.defaultProps_), staticProps);
+    }
```

Comment:
I added this condition for consistency with below, but I don't think it's needed in either case?

<img src="https://user-images.githubusercontent.com/254946/94179252-1963d300-fe51-11ea-823d-13cd9e2cef25.png" alt="image" width="300">

Reply:
Yeah. It's not necessary. We can even combine them into one:

```
Object.assign(this.defaultProps_, staticProps, iniProps);
```